cluster/gce/gci/configure-helper.sh

```diff
@@ -1688,6 +1688,7 @@ function start-kube-proxy {
 function prepare-etcd-manifest {
   local host_name=${ETCD_HOSTNAME:-$(hostname -s)}
   local -r host_ip=$(${PYTHON} -c "import socket;print(socket.gethostbyname(\"${host_name}\"))")
+  local -r host_primary_ip=$(hostname -i)
```

Comment:
what is the difference between host_ip and host_primary_ip? Do we need both?

Reply:
In regional clusters, host_ip is an alias IP in range 10.127.X.X, and host_primary_ip is primary IP in range 10.128.X.X. The former is only used for etcd peering. In zonal clusters, the two are the same.

Reply:
We need to add that as a comment here.

Reply:
Added documentation to explain the difference between two variables.